api-reference/10 UI Components/dxCalendar/1 Configuration/dateSerializationFormat.md

```diff
@@ -5,22 +5,40 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the date-time value serialization format. Use it only if you do not specify the [value](/api-reference/10%20UI%20Components/dxCalendar/1%20Configuration/value.md '/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#value') at design time.
+Specifies the date-time value serialization format.
 
 ---
-Without a value, the UI component cannot detect its format. In this case, specify the **dateSerializationFormat** property that supports the following formats:
+Date-time serialization involves converting date-time objects into a string format for storage or transmission. To ensure proper format detection, specify this property.
```

Comment:
I just want to use "conversion" instead of "converting". 

```suggestion
Date-time serialization involves date-time value conversion into a string format for storage or transmission. To ensure proper format detection, specify this property.
```